app/src/main/java/fr/free/nrw/commons/Urls.kt

```diff
@@ -0,0 +1,14 @@
+package fr.free.nrw.commons
+
+internal object Urls {
+    val NEW_ISSUE_URL = "https://github.com/commons-app/apps-android-commons/issues"
```

Comment:
```suggestion
    const val NEW_ISSUE_URL = "https://github.com/commons-app/apps-android-commons/issues"
```

(and all of these in the file)

These are constants, and using `const` allows them to be referenced without an instance (like `static` in Java) - e.g. instead of `Urls.INSTANCE.getTRANSLATE_WIKI_URL()` you can just use `Urls.TRANSLATE_WIKI_URL`.

Reply:
@domdomegg Thanks for the suggestion!